src/config.ts

```diff
@@ -40,6 +40,12 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
       setting: "",
       help: "Print the current version and exit.",
     }),
+    richbitch: Args.flag({
```

Comment:
As Katarn has mentioned in the discord, we should change this to something that isn't as explicit as this. Perhaps `candydish` or `redtooth` or something else descriptive that isn't too lengthy.

Reply:
:(